Resources/public/js/models/ez-contentmodel.js

```diff
@@ -316,6 +317,17 @@ YUI.add('ez-contentmodel', function (Y) {
              */
             relations: {
                 value: {}
+            },
+
+            /**
+             * The version info of the content
+             *
+             * @attribute versionInfo
+             * @type Object
+             * @default {}
+             */
+            versionInfo: {
```

Comment:
I would store the whole version instead of just the version info. You should also call it `currentVersion`